SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitPaymentAuthRequestCallback.kt

```diff
@@ -0,0 +1,15 @@
+package com.braintreepayments.api
+
+/**
+ * Callback for receiving result of
+ * [SEPADirectDebitClient.createPaymentAuthRequest].
+ */
+fun interface SEPADirectDebitPaymentAuthRequestCallback {
+    /**
+     * @param paymentAuthRequest the result of the SEPA create mandate call. If a nonce is present,
+     * no web-based mandate is required. If a nonce is not present, you
+     * must trigger the web-based mandate flow via
+     * [SEPADirectDebitLauncher.launch]
+     */
+    fun onResult(paymentAuthRequest: SEPADirectDebitPaymentAuthRequest)
```

Comment:
do we send a request on a `onResult` callback? 🤔

Reply:
That's actually a good call - we want to update all these callbacks from `onResult` to `onWhateverItReturns` - fixed this one